src/main/java/com/example/analytics/model/coinMarketCap/CryptoCurrencyData.java

```diff
@@ -0,0 +1,20 @@
+package com.example.analytics.model.coinMarketCap;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+public class CryptoCurrencyData {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String symbol;
```

Comment:
Let's call it **ticker**, not **symbol**

Reply:
changed from symbol to ticker, and also all other fields and sources were changed